backend/assessments/2/src/main/java/com/kdu/assessment/entities/Order.java

```diff
@@ -0,0 +1,37 @@
+package com.kdu.assessment.entities;
+
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+import java.util.Map;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "orders")
+public class Order extends BaseEntity{
```

Comment:
Just minor nitpicking - ideally lets have a parameter for soft delete as you might to show cancelled orders for a certain time period (not cutting marks)